steam/manifest.py

```diff
@@ -949,7 +949,8 @@ def __init__(
 
         self.language_support = {
             Language.from_str(language): support
-            for language, support in common.get("supported_languages", MultiDict()).items()
+            for language, support in (common.get("supported_languages") or MultiDict()).items()
+            if isinstance(common.get("supported_languages"), dict)
```

Comment:
Use not isinstance(x, str) (dict won't work because they aren't dicts) and update the type definition to reflect this can be a string occasionally

Reply:
updated